libs/wingc/src/type_check.rs

```diff
@@ -1740,25 +1825,50 @@ impl<'a> TypeChecker<'a> {
 		}
 	}
 
-	fn add_module_to_env(&mut self, env: &mut SymbolEnv, module_name: String, statement_idx: usize) {
-		// TODO Hack: treat "cloud" or "std" as "_ in wingsdk" until I figure out the path issue
-		if module_name == "cloud" || module_name == "fs" || module_name == WINGSDK_STD_MODULE {
-			let mut wingii_types = wingii::type_system::TypeSystem::new();
+	fn add_module_to_env(
+		&mut self,
+		env: &mut SymbolEnv,
+		library_name: String,
+		namespace_filter: Vec<String>,
+		identifier: &Symbol,
```

Comment:
Semantic inconsistency -- `identifier` was called `alias` before (I think):



```suggestion
		alias: &Symbol,
```

Reply:
Updated